### DISCUSSION TITLE
blog/tcp-performance/

### DISCUSSION THREAD

**Discussion by giscus** [Jitesh117/blog_comments#12] [Q&A]
# blog/tcp-performance/

HTTP is the language of the internet. And it is layered directly on top of TCP, so the performance of HTTP calls depends directly on the performance of the underlying TCP.
This article highlights some significant performance considerations of TCP connections. You’ll better appreciate HTTP’s connection optimization features after understanding these basic performance characteristics of TCP.
HTTP Transaction Delays There are several possible causes of delay in an HTTP transaction:

https://jitesh117.github.io/blog/tcp-performance/

**Comment by melqtx** [comment]
interesting also the redesign looks awesome
